packages/wallet-manager/src/WalletManagerController.ts

```diff
@@ -0,0 +1,162 @@
+import { ReactiveController, ReactiveControllerHost } from 'lit';
+import { EvmWallet, SubstrateWallet } from '.';
+import { Web3Provider } from '@ethersproject/providers';
+import { ApiPromise } from '@polkadot/api';
+import { AddChain } from './types';
+import { IWalletManagerController } from './interfaces';
+import { Signer } from '@ethersproject/abstract-signer';
+
+export class WalletManagerController implements IWalletManagerController {
+  host: ReactiveControllerHost;
+  evmWallet?: EvmWallet;
+  substrateWallet?: SubstrateWallet;
+  account?: string;
+  substrateAccount?: string;
+
+  constructor(host: ReactiveControllerHost) {
+    (this.host = host).addController(this as ReactiveController);
```

Comment:
the problem here was that `ReactiveController` is a weak type, and since I wasn't implementing any of the methods from that interface, I needed to do the type assertion. Solved